components/applications-service/cmd/applications-load-gen/commands/root.go

```diff
@@ -0,0 +1,45 @@
+package commands
+
+import (
+	"os"
+
+	"github.com/spf13/cobra"
+)
+
+var profileFile string
+var useDefaultProfile bool
```

Comment:
I decided to use a struct for these since it reduces the number of variable names in the package's namespace and makes it a bit more explicit which things are which.